index.html

```diff
@@ -1,12 +1,18 @@
 <html>
   <body>
-    <button onclick="todo()">Count by 5</button>
+    <button id="a" onclick="todo()">Count by 5</button>
```

Comment:
The ID should be more meaningful than `a`.

Reply:
Also, if you're attaching the click handler below, you should remove the `onclick="todo()"` in this line.